videomaker/sass/templates/_footer.scss

```diff
@@ -0,0 +1,8 @@
+footer > .wp-block-group {
+	align-items: center;
+	justify-content: space-between; // Needed until we have a way to set this via block attributes
+	padding: var(--wp--custom--gap--horizontal);
```

Comment:
Can we set this in the block attributes?

Reply:
Yes good call, also I realized I probably just need left and right, not top and bottom because the vertical spacing is already provided by the group block.

Reply:
Actually based on https://github.com/Automattic/themes/pull/4698 I think we will need the top and bottom padding, I added it back in the block attributes.

Reply:
I think that's going to be removed in https://github.com/Automattic/themes/pull/4698